### ISSUE TITLE
Better mps-youtube integration

### ISSUE THREAD

**Issue by woorst** [lwilletts/mpvc#24]
Might I suggest a better way to use `mpvc` and `mps-youtube`?

`set download_command mpvc add https://youtu.be/%i#%f -- --vid=no`

Then add tracks in mpsyt using, for example: `da 1-5`

Advantages:
- When using `mpvc current` you actually see info for the track playing instead of a crazy googlevideo.com link.
- When using `mpvc -i` the filename will include more interesting track information which can better parsed in the script.

**Comment by wildefyr** [comment]
Haven't got a way to test this currently, can you confirm for me this method works?

**Comment by lwilletts** [comment]
Just done some very quick testing on this: I like it, but the method above seems to 'lock' mps-youtube up and the terminal window has to stay open for mpvc to continue to be fed the stream, although this is true of the add method as well. A solution to this might just be to download the files fully through mpvc using youtube-dl into a temporary folder, then play them back normally using mpvc.

**Comment by lwilletts** [comment]
Very on the fence about this, on one hand it is some great functionality to add. On the other, the problem this issue is caused by is mps-yt. Not mpvc. mpvc recognises urls passed to it and adds them to the playlist absolutely fine, mps-yt however decides to lock up using your method. On my third hand, player=mpvc, playerargs=add mps-yt seems to think it fails adding and calls mpvc add URL a further two times. 

So in conclusion, mps-yt is dumb. This is an upstream issue, find a better youtube browser that is sane.
